config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "hello#index"
+  resources :users, only: [:show] do
```

Comment:
- [ ] Kindly add a route for the users `index` action. It should read `:users, only: [:show, :index]`